src/server/connection.rs

```diff
@@ -14,6 +14,7 @@ use crate::platform::linux_desktop_manager;
 use crate::platform::WallPaperRemover;
 #[cfg(windows)]
 use crate::portable_service::client as portable_client;
+use crate::ui_interface::{get_local_option, set_local_option};
```

Comment:
server/connection.rs does nothing with the UI, so you can not use any thing of ui_interface here.